test/test.js

```diff
@@ -24,7 +24,7 @@ describe('telnet', function () {
         done();
       });
     });
-
+    /*
```

Comment:
This mocha test passes locally, but fails on Travis. If I comment this section, it passes on Travis... Needs some investigation. As it is just clean up, do you see and issue in commenting it in the mean time?